test/NameRegistry.t.sol

```diff
@@ -235,6 +235,51 @@ contract NameRegistryTest is Test {
         nameRegistry.makeCommit(commitHash);
     }
 
+    /*//////////////////////////////////////////////////////////////
+                            VIEW FUNCTIONS
+    //////////////////////////////////////////////////////////////*/
```

Comment:
idea: we could make a harness contract, but probably not worth it for just four helpers